x-pack/plugins/xpack_main/server/lib/register_oss_features.ts

```diff
@@ -119,6 +119,10 @@ const kibanaFeatures: Feature[] = [
         ui: [],
       },
     },
+    privilegesTooltip: i18n.translate('dev_tools.privileges.tooltip', {
```

Comment:
I'm guessing the linter will complain that this doesn't start with `xpack.main` or similar

Reply:
Good call, I keep forgetting about that...